app/scripts/directives/overview/serviceInstanceRow.js

```diff
@@ -44,7 +44,15 @@ function ServiceInstanceRow($filter, DataService, rowMethods, $uibModal) {
     row.notifications = rowMethods.getNotifications(row.apiObject, row.state);
     row.displayName = getDisplayName();
     row.description = getDescription();
-    row.instanceBindings = getBindings();
+  };
+
+  var prevBindings = [];
+  row.$doCheck = function() {
+    var bindings = getBindings();
+    if(prevBindings.length !== bindings.length) {
```

Comment:
@jwforres this just hits adding the additional binding row, I'm working on looking at the statuses now.

@spadgett i can do that, assuming that we would want something like this then:
```html 
<service-instance-row
  api-object="serviceInstance"
  bindings="bindingsByRef[serviceInstance.metadata.name]"></service-instance-row>
```